core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/integration_test.go

```diff
@@ -508,6 +509,7 @@ func setupDependencies(t *testing.T, db *sqlx.DB, backend *backends.SimulatedBac
 	}
 	ht := headtracker.NewSimulatedHeadTracker(ethClient, lpOpts.UseFinalityTag, lpOpts.FinalityDepth)
 	lp := logpoller.NewLogPoller(lorm, ethClient, pollerLggr, ht, lpOpts)
+	servicetest.Run(t, lp)
```

Comment:
Not sure I understand. If it was being started somewhere else and not stopped, are we starting it twice now? Shouldn't we remove wherever else it was being started?

Reply:
I'm still hacking on this one. I'll switch it back to draft.

The leak suggests the test is not blocking on Close()